File: exerc_fund_comp/Fundamentos Lista 6/ex5.py
vet1 = [0] * 10
vet2 = [0] * 10
vetInter = []

for i in range(len(vet1)):
    vet1[i] = int(input("Informe um valor inteiro para o 1º vetor: "))

for i in range(len(vet2)):
    vet2[i] = int(input("Informe um valor inteiro para o 2º vetor: "))

for i in range(len(vet1)):
    for j in range(len(vet2)):
        if vet1[i] == vet2[j]:
            vetInter.append(vet2[j])

print(vetInter)

File: exerc_aed1/Lista de Exercícios 0/ex5.c
#include <stdio.h>

int main()
{
    int n = 0;
    do
    {
        printf("Informe o valor de N: ");
        scanf("%d", &n);
    } while (n <= 0);

    int i;
    float s = 0;
    for (i = 1; i <= n; i++)
    {
        s += 1.0 / i;
    }

    printf("\nSoma: %.2f\n", s);
    return 0;
}
